### PR TITLE
Pro 2487

### PR DESCRIPTION
**Pull request by michelleinez**

### CODE REVIEW

File: src/ui/screens/RegistrationForm.tsx [hunk 1]
```diff
@@ -67,54 +78,23 @@ export default class RegistrationForm extends React.Component<Props, State> {
...
-                  value={this.props.credentialCreationData.nationalId}
-                  fullWidth
-                  name="nationalId"
-                  id="nationalId"
-                  placeholder="National ID"
-                />
-              </Grid>
+              {_.keys(this.props.credentialCreationData).map(
+                (field: any, idx: any) => {
+                  if (field != "faceImage") {
```
Comment (horationunez) [inline src/ui/screens/RegistrationForm.tsx:83]: This isn't a blocker for now but something to keep in mind for the future. We don't want to rely on the field name as much as a configuration that indicates the type of the field.

**Comment by horationunez** [comment]
@michelleinez you might have to review the linter errors with `eslint src` since the compilation is taking warnings as errors. We want to enforce this warnings since it ensures we don't have any feature half done (like an unused variable).

**Review by horationunez — CHANGES_REQUESTED** [review]
Please review the linter warnings.

File: src/ui/screens/RegistrationForm.tsx (hunk 1 above)
Comment (michelleinez) [reply src/ui/screens/RegistrationForm.tsx]: Okay, I addressed this issue.

**Pull request merged**